apphub/instance_segmentation/solov2/solov2_torch.py

```diff
@@ -284,22 +285,6 @@ def remove_empty_gt(masks, bboxes):
         non_empty_mask = np.sum(masks.reshape(num_objects, -1), axis=1) > 0
         return masks[non_empty_mask], bboxes[non_empty_mask]
 
-
-class NormalizePermute(TensorOp):
-    def __init__(self, inputs, outputs, mean, std, mode=None):
-        super().__init__(inputs=inputs, outputs=outputs, mode=mode)
-        self.mean = to_tensor(np.array(mean, dtype="float32"), "torch")
-        self.std = to_tensor(np.array(std, dtype="float32"), "torch")
-
-    def build(self, framework, device):
-        self.mean = self.mean.to(device)
-        self.std = self.std.to(device)
-
-    def forward(self, data, state):
-        data = (data / 255 - self.mean) / self.std
-        return data.permute((0, 3, 1, 2))  # channel first
```

Comment:
you'll need to have a permute op in the pipeline now instead right?

Reply:
The current solo torch implementation doesn't contain the permute option. But if that's the intension, than will use that instead of it been a part of normalize.

Reply:
Create a new permute operation, to perform this instead of direct permute.